src/Writers/HTMLWriter.jl

```diff
@@ -1748,7 +1749,17 @@ function mdconvert(d::Dict{MIME,Any}, parent; kwargs...)
     if haskey(d, MIME"text/html"())
         out = Documents.RawHTML(d[MIME"text/html"()])
     elseif haskey(d, MIME"image/svg+xml"())
-        out = Documents.RawHTML(d[MIME"image/svg+xml"()])
+        svg = d[MIME"image/svg+xml"()]
+        # the xmlns attribute has to be present for data:image/svg+xml
+        # to work (https://stackoverflow.com/questions/18467982)
+        # URIs.escapeuri is needed to replace all special characters from the SVG string
+        # which could break the HTML
+        out = if occursin("xmlns=", svg)
+            escaped_svg = URIs.escapeuri(svg)
```

Comment:
Would base-64 encoding work as well? This way we could avoid the dependency.

Reply:
I'm not sure, I can test it. Maybe a function doing the escaping could be copied as well, it can't be that complicated.

Reply:
at least this page https://css-tricks.com/probably-dont-base64-svg/ recommends not to use base64, because of 33% larger filesize